### PR TITLE
tensorflow.bzl: strip bazel version number to work with patch bazel release

### PR DESCRIPTION
**Pull request by meteorcloudy**
The newest bazel version number is 0.2.2b, when need to strip 2b

### CODE REVIEW

File: tensorflow/tensorflow.bzl [hunk 1]
```diff
@@ -12,7 +12,7 @@ def _parse_bazel_version(bazel_version):
   # Turn "release" into a tuple of integers
   version_tuple = ()
   for number in parts[0].split('.'):
-    version_tuple += (int(number),)
+    version_tuple += (int(number.strip('abcdefghij')),)
```
Comment (vrv) [inline tensorflow/tensorflow.bzl]: can you replace 'int' with 'str' here ?

version_tuple += (str(number),)

We'll compare strings lexicographically, which will probably solve your problem here.

Ideally we'd all use semantic versioning conventions though.

**Comment by vrv** [comment]
There are probably better ways to do this, but in the interest of unbreaking people, this seems fine.  Going to test and merge if successful.

**Comment by vrv** [comment]
@tensorflow-jenkins test this please

**Comment by yvirin** [comment]
hurray

**Pull request merged**